doc/user/data/sql_funcs.yml

```diff
@@ -388,6 +388,12 @@
       &bull; Side: `BOTH`<br/>
       &bull; `'c'`: `' '` (space)"
 
+  - signature: 'try_parse_monotonic_iso8601_timestamp(s: str) -> timestamp'
```

Comment:
Does it return`timestamp`? Would it be more future-proof to let it return `timestamptz` for later extensions to support other time zones than `Z`?

Reply:
I think we'd just add a `try_parse_monotonic_iso8601_timestamptz` function in that case!